### ISSUE TITLE
Update on the code release

### ISSUE THREAD

**Issue by julien-blanchon** [alex4727/MotionStream#1]
Hello, amazing work !

Could you update here (comment and close this issue) when the code is release, so I can get a notification ?

**Comment by alex4727** [comment]
Sure! I'll leave comments here too :)

**Comment by venturaEffect** [comment]
Me too.

**Comment by mingzhe11** [comment]
Me too.

**Comment by filipzdelar** [comment]
Me too.

**Comment by antithing** [comment]
Amazing work! Looking forward to the release.

**Comment by hey0chen** [comment]
Me too.

**Comment by Caselles** [comment]
Same

**Comment by yupengchengg147** [comment]
same

**Comment by pkulium** [comment]
same

**Comment by zyinghua** [comment]
same

**Comment by MariPS** [comment]
Eagerly awaiting for the update!

**Comment by snitchjinx** [comment]
same

**Comment by johndpope** [comment]
you might be surprised by how far you can get with claude code 4.6. most recently  I "reverse" (more accurately forward) engineered the seed dance paper - https://github.com/johndpope/OmniTransfer-hack 
if you were to point claude at that paper - you may end up with something similiar on your own.
we have the losses - models - need to throw some compute + training data - but pretty confident it will work properly.   N.B. that omnitransfer was based on wan2.1 - but the world got LTX2 - + teacache - so choosing that video model maybe stronger until next big video drops. 

even if this code drops - it will be out of date video model - probably no training scripts. wan 2.1 seems EOL.

**Comment by lishiqianhugh** [comment]
Looking forward to the release!

**Comment by johndpope** [comment]
related papers dropping with code
https://spherelab.ai/diagdistill/
https://github.com/quanhaol/FlashMotion
